skymap_scanner/utils/extract_json_message.py

```diff
@@ -50,34 +51,60 @@ def extract_GCD_diff_base_filename(frame_packet):
 
     return GCD_diff_base_filename
 
+def __get_pulses_name(event_dict: dict):
```

Comment:
Here and a few other new functions start with `__`. What's the reasoning to no use a single `_`? I think it's technically okay, but I believe python does some strange magic when class methods/attributes are prefixed with `__`.

Reply:
I think we use underscores here to mark a function that has been split out a previously existing function, due to the latter being too complicate. In this improvised naming convention  `__g` is a function that splits some code out of `_f` which was originally split out of some other function. Not sure whether I was the first to use the double underscore here or not.

However this is meant to be somewhat temporary, like: "I have split out some code that was previously there but probably we should review this more in depth". But I agree, we can avoid double underscores just to be safe.

Reply:
Now updated all double underscores to single underscore.

Reply:
> However this is meant to be somewhat temporary, like: "I have split out some code that was previously there but probably we should review this more in depth". But I agree, we can avoid double underscores just to be safe.

That explains it until the original intention is forgotten :smile: 

Tangent: I think there's nothing wrong with very long function names as long as it's accurately descriptive. But this _is_ a controversial opinion in the dev world